### PR TITLE
Convert testJob.yaml to daily Cronjob & Exclude Validated Partners PR's from GH actions workflow runs

### PR DESCRIPTION
**Pull request by mikemcd3912**
*Issue #, if available:* 
Sample Tester manifest is Job, should be CronJob #142
GH Action Issue (Failing) for Validated Partners Auto PR. #182
GH Actions Issue with an existing Partner addon. #163

*Description of changes:*
Updated Sample tester job to daily Cronjob. Updated Cronjob file was deployed and tested to conform intended function prior to PR creation
Updated GH actions run files to exclude Validated Partner PR's (PR's including only changes to the Validated Partner folder) or updates to the .github folder from triggering the automation run
Updated GH Actions new PR file to fix subdirectory dependance between file processing. Workflow is re-ordered to move updated files over first and perform NS file check/PR configmap generation for each file's subdirectory 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@mikemcd3912 You should also update the test job in each of the deployment model (BM, vSphere, Snow, Localcluster, Nutanix) tester sample folders eg. https://github.com/aws-samples/eks-anywhere-addons/blob/main/eks-anywhere-baremetal/Testers/Sample/testJob.yaml

**Comment by mikemcd3912** [comment]
@elamaran11 Test job in each of the deployment model tester sample folders are now updated

File: .github/workflows/new-pull-request.yaml [hunk 1]
```diff
@@ -4,6 +4,9 @@ on:
   pull_request_target:
     branches: [main]
     types: [opened, reopened, synchronize]
+    paths-ignore: 
+     - 'Validated_Partners/**'
+     - '.github/**'
```
Comment (elamaran11) [inline .github/workflows/new-pull-request.yaml:9]: What is this? Why are we doing this ignore?

File: .github/workflows/close-pull-request.yaml [hunk 2]
```diff
@@ -3,6 +3,9 @@ on:
   pull_request_target:
     branches: [main]
     types: [closed]
+    paths-ignore: 
+     - 'Validated_Partners/**'
+     - '.github/**'
```
Comment (elamaran11) [inline .github/workflows/close-pull-request.yaml:8]: What is this? Why are we doing this ignore?

**Review by elamaran11 — COMMENTED** [review]
@mikemcd3912 Have a minor question, rest looks good. Make sure you test it

File: .github/workflows/new-pull-request.yaml (hunk 1 above)
Comment (mikemcd3912) [reply .github/workflows/new-pull-request.yaml:9]: This ignore is to stop the run from triggering on PR's where the only files changed came from one of these two paths  GH actions updates or the automated Validated_Partners runs. If other files not found in these folders are changed in a PR the action will still run

**Comment by mikemcd3912** [comment]
Updated GH Actions new PR file to fix subdirectory dependance between file processing. Workflow is re-ordered to move updated files over first and perform NS file check/PR configmap generation for each file's subdirectory (Addressing issue #163)

**Review by elamaran11 — APPROVED** [review]
LGTM. Nice work Mike. Still we are missing PRs from partners with existing SWs. Im expecting a new PR for the same

**Comment by elamaran11** [comment]
@mikemcd3912 Changes look good but why is the GH Action failing. Is this a boundary condition you are missing to handle.

**Comment by mikemcd3912** [comment]
The Github Action is failing because we haven't merged these changes and it's attempting to run based on the old action file - Under the new ignorePaths this PR wouldn't trigger a GH action run since we're just editing GH action files

**Comment by elamaran11** [comment]
Sounds good, We can merge this but if further GH action fails we have to revert this PR.

**Pull request merged**